app/Http/Requests/MemberInviteRequest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class MemberInviteRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, mixed>
+     */
+    public function rules()
+    {
+        return [
+            'name' => 'required',
+            'email' => 'required | email',
+            'role_id' => 'required | integer',
+            'vendor_employee_id' => 'required | integer'
+        ];
+    }
+
+    /*
+     * Custom message for validation
+     *
+     * @return array
+     * */
+    public function messages()
```

Comment:
@Babita3331 If there are no requirements to have custom validation messages, not really necessary to define custom messages. However do check where these default validation messages are coming from.